frontend-public/src/components/common/UploadedFilesList.js

```diff
@@ -30,12 +28,8 @@ export class UploadedFilesList extends React.Component {
   };
 
   canUnlink = (selectedDoc) => {
-    const docStatus = this.props.expectedDocumentStatusOptions.find(
-      (status) => status.value === selectedDoc.exp_document_status_guid
-    );
     const dueDateCheck = moment(selectedDoc.due_date, "YYYY-MM-DD") > moment();
-    const statusCheck =
-      docStatus.label === "Not Received" || docStatus.label === "Received / Pending Review";
+    const statusCheck = selectedDoc.exp_document_status.exp_document_status_code in ['MIA', 'PRE']
```

Comment:
`in` operator is for objects, not arrays
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in

My understanding is that this will always return `false`. You'll be looking for `Array.prototype.includes()`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Reply:
My mind was still in python mode 😅

Reply:
Good catch.